### PR TITLE
Import and rework the paranoia levels blog post

### PR DESCRIPTION
**Pull request by RedXanadu**
This PR imports and reworks the content from the recent paranoia levels blog post.

### CODE REVIEW

File: content/configuring/paranoia_levels.md [hunk 1]
```diff
@@ -4,4 +4,87 @@ chapter: false
...
+
+A higher paranoia level makes it harder for an attacker to go undetected. Yet this comes at the cost of more false positives: more false alarms. That's the downside to running a rule set that detects almost everything: your business / service / web application is also disrupted.
+
+When false positives occur they need to be tuned away. In ModSecurity parlance: rule exclusions need to be written. A rule exclusion is a rule that disables another rule, either disabled completely or disabled partially only for certain parameters or for certain URIs. This means **the rule set remains intact** yet the CRS installation is no longer affected by the false positives.
+
+{{% notice note %}}
+Depending on the complexity of the service (web application) in question and on the paranoia level, the process of writing rule exclusions can be a *substantial* amount of work.
+
+This page wont explore the problem of handling false positives further: for more information on this topic, see the appropriate chapter or refer to the [tutorials at netnea.com](https://www.netnea.com/cms/apache-tutorials/).
```
Comment (dune73) [inline content/configuring/paranoia_levels.md:24]: Typo: wont

**Comment by dune73** [comment]
Thank you for reworking my text. I saw nothing amiss outside of said typo.

I realised it's relatively hard for me to gauge the display of the new documentation based on a markdown. So there might be issues that only appear once we merge. A problem that will go away with more experience with the new documentation for me.

**Comment by RedXanadu** [comment]
@dune73 Agreed. Especially with complex pages that use diagrams, admonitions, and other funky Markdown/Hugo features, the raw files can become difficult to read.

For people reviewing documentation content in future PRs, there are two approaches that may make it a bit easier:
- Clone the branch to be merged and then run `hugo serve`. You can then see how the end result will look (this requires following the README instructions to grab the Hugo executable.)
- Navigate to the branch to be merged on GitHub and click on the Markdown file in question. GitHub does its best to present Markdown files, although a few bits will not show up (like Hugo-specific features, e.g. admonitions).

**Comment by dune73** [comment]
Merging now.

**Pull request merged**